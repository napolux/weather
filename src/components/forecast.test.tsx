import React from 'react';
import Forecasts from './forecast';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <Forecasts />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});