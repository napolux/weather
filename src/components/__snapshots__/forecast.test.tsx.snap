// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui container"
>
  <div
    className="ui cards"
  />
</div>
`;
